Xamarin.Forms.Core/Page.cs

```diff
@@ -311,7 +311,8 @@ void IPageController.SendAppearing()
 				handler(this, EventArgs.Empty);
 
 			var pageContainer = this as IPageContainer<Page>;
-			((IPageController)pageContainer?.CurrentPage)?.SendAppearing();
+			if (!(this is TabbedPage))
```

Comment:
this also made CarouselPage send the appearing event.. i wonder if this reverts that.